src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -109,19 +130,24 @@ def load_covariates(self, scenario: ScenarioSpecification, location_ids: List[in
         covariate_data = reduce(lambda x, y: x.merge(y, left_index=True, right_index=True), covariate_data)
         return covariate_data.reset_index()
 
-    def load_regression_coefficients(self, draw_id: int) -> pd.DataFrame:
-        file = self.regression_paths.get_coefficient_file(draw_id)
-        return pd.read_csv(file)
-
-    def load_beta_params(self, draw_id: int) -> pd.DataFrame:
-        beta_params_file = self.regression_paths.get_beta_param_file(draw_id)
-        df = pd.read_csv(beta_params_file)
+    def load_beta_params(self, draw_id: int) -> Dict[str, float]:
+        df = self.regression_marshall.load(key=MKeys.parameter(draw_id=draw_id))
         return df.set_index('params')['values'].to_dict()
 
-    def save_forecasts(self, forecasts: pd.DataFrame, scenario: str, draw_id: int):
-        forecast_path = self.forecast_paths.get_component_draws_path(draw_id, scenario)
-        forecasts.to_csv(forecast_path, index=False)
+    def save_components(self, forecasts: pd.DataFrame, scenario: str, draw_id: int):
+        self.forecast_marshall.dump(forecasts, key=MKeys.components(scenario=scenario, draw_id=draw_id))
+
+    def load_components(self, scenario: str, draw_id: int) -> pd.DataFrame:
+        return self.forecast_marshall.load(key=MKeys.components(scenario=scenario, draw_id=draw_id))
```

Comment:
unused; added for symmetry with save_components (someone has to load them eventually, right?)